packer/install-dependencies.sh

```diff
@@ -360,6 +360,7 @@ echo "  - busybox" >> ${RELEASE_NOTES_FILEPATH}
 
 # TODO: fetch supported k8s versions from an aks-engine command instead of hardcoding them here
 K8S_VERSIONS="
+1.14.3
 1.14.2
```

Comment:
Maybe we can remove 1.14.2 since it's disabled (and we most likely won't re-enable it)